src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
You have only specified one font-family. It's a good practice to specify a list of font-families as a fallback, in case the first specified font is not available. You can add more generic font families like 'Arial', 'sans-serif' etc.